packages/web-components/src/components/tabs-extended/tab.ts

```diff
@@ -0,0 +1,83 @@
+/**
+ * @license
+ *
+ * Copyright IBM Corp. 2020
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { html, customElement, property, LitElement, internalProperty } from 'lit-element';
+import ddsSettings from '@carbon/ibmdotcom-utilities/es/utilities/settings/settings.js';
+import styles from './tabs-extended.scss';
+import StableSelectorMixin from '../../globals/mixins/stable-selector';
+
+const { stablePrefix: ddsPrefix } = ddsSettings;
+
+/**
+ * The copy content of content item horizontal.
+ *
+ * @element dds-tab
+ */
+@customElement(`${ddsPrefix}-tab`)
+class DDSTab extends StableSelectorMixin(LitElement) {
+  /**
+   * Defines label of the tab.
+   */
+  @property({ reflect: true })
+  label = null;
+
+  /**
+   * Defines title of the tab.
+   */
+  @property({ reflect: true })
+  title = null;
```

Comment:
This line may yield to type inference to `null`, though it looks like a string. `title = ''` may be more desirable?